### ISSUE TITLE
Update S3 files collection after image deleted

### ISSUE THREAD

**Issue by ansonla3** [Lepozepo/S3#108]
How is it possible to update the S3 files collection when delete method is called?

Since I tried to delete an image that S3 shown was removed, S3 files collection still has the file exist.

Please help.

**Comment by Lepozepo** [comment]
Hey! After calling `S3.delete(path,cb)`, you should be able to use the `cb` function to do something like `S3.collection.findOne({relative_url:path})`. This should be more intuitive though, I've been wanting to rewrite this package for a while now. I have a project where I know I will use it so an update is on it's way to simplify these things.

**Comment by ansonla3** [comment]
yes, this solved. Thanks very much.

I got another problem. I am using Meteor 1.3.1, React 0.15 right now. 

Before I was successfully delete the image , but after I installed the webpack:webpack. Then, when I delete the image on the client slide, I got this error:

      S3.delete(selectedImage,function(error, result) {
        console.log("remove result: ", result);
      });

**Browser**
Uncaught TypeError: Cannot read property 'delete' of undefined


Thanks very much
